sagenb/notebook/cell.py

```diff
@@ -131,7 +131,7 @@ def __cmp__(self, right):
             sage: [C1 == C2, C1 == C3, C2 == C3]
             [True, True, True]
         """
-        return cmp(self.id(), right.id())
+        return self.id() < right.id()
```

Comment:
Oof, I already merged this but I may have been too quick.  Although it seems this is only needed for sorting (and hence `__lt__` being sufficient) this seems like it will break the above doctests that also check for equality.

I think for backwards compatibility you should add back in `__eq__` as well even if it's not directly used within the module.